### DISCUSSION TITLE
problem with meter

### DISCUSSION THREAD

**Discussion by famko09** [israel-dryer/ttkbootstrap#373] [Q&A]
i have a problem with a meter, 

1)
when I change the value with the configure option, there is sometimes an empty square visible in the center, depending on how characters the value is. It looks like it's an empty label from the subtext option.

2)
When the value is below 1, there is no indicator showing. Is this possible to let the indicator show value between 0-1?

`cpu_1_speed = ttk.Meter(master=cpu_1_frame,metersize=100,metertype="semi",stripethickness=7,interactive=False,bootstyle='success',amounttotal=2)
cpu_1_speed.configure(amountused = cpu_1_speed_value)
`
![meter](https://user-images.githubusercontent.com/74816281/202115340-ae625c0b-0620-440a-984b-346d2a03aa4c.jpg)
![meter2](https://user-images.githubusercontent.com/74816281/202115351-939e368d-30e0-4fd3-9584-a5ca0c99876f.jpg)

**Comment by Krzych0409** [comment]
I have the same problem.

**Comment by CodingLama** [comment]
Maybe this would better fit the Bug category than in here, I guess.

**Comment by TheDemonGuard** [comment]
@famko09 
It seems the issue is resolved with the fix #704 
Can you try with the latest version again.

> i have a problem with a meter,
> 
> 
> when I change the value with the configure option, there is sometimes an empty square visible in the center, depending on how characters the value is. It looks like it's an empty label from the subtext option.
> 
> 
> When the value is below 1, there is no indicator showing. Is this possible to let the indicator show value between 0-1?
> 
> `cpu_1_speed = ttk.Meter(master=cpu_1_frame,metersize=100,metertype="semi",stripethickness=7,interactive=False,bootstyle='success',amounttotal=2) cpu_1_speed.configure(amountused = cpu_1_speed_value) ` ![meter](https://user-images.githubusercontent.com/74816281/202115340-ae625c0b-0620-440a-984b-346d2a03aa4c.jpg) ![meter2](https://user-images.githubusercontent.com/74816281/202115351-939e368d-30e0-4fd3-9584-a5ca0c99876f.jpg)
